Nelwell/lmn
# Allow user to upload images with each note.

Users should now be able to upload photos with each note that they create. 

To test: 
1. Create at least one test note and corresponding show/venue in admin console
2. Go to "Notes" in running application
3. Each note in the list will have a link to the show it's attached to - select one of the links
4. Scroll to the bottom of the screen and select "add your own notes" 
5. A form will display, asking for a title, text, and image - test to see that the selected image displays with the note after you've 
added it

@Nelwell @abdiawali @PickleEric
